### ISSUE TITLE
nimble lock computes different hash on Windows than Linux/macOS

### ISSUE THREAD

**Issue by iffy** [jangko/msgpack4nim#81]
I filed this issue with nimble: https://github.com/nim-lang/nimble/issues/1188

I don't really know what the fix is. The trouble is that this repo's `.gitattributes` file causes a different hash to be computed on Windows vs Linux/macOS, but I can understand why you want that file in this repo. Do you have a suggestion?

**Comment by jangko** [comment]
because of EOL conversion?

**Comment by iffy** [comment]
Yes, nimble computes a hash of the directory of installed packages for use in a lock file. On Windows, because of the EOL conversion, the hashes are different than those on Linux/macOS.

**Comment by jangko** [comment]
actually only one file is problematic `tests/basic.mp` should be treated as binary instead of text.
